cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.md

```diff
@@ -387,6 +392,13 @@ For Angular (2+) remember to build Angular templates with `-prod` parameter (`ng
 
 And also remember to update your framework to the newest version, with all possible bug fixes, as soon as possible. 
 
+## Bonus Rule \#6: Use Trusted Types
```

Comment:
I'm a little hesitant about this, because it seems so immature as as feature. From what I can see, Google announced this in February 2019 (requiring either a runtime argument or chrome://flags to enable it), and there's been no updates on it since then.

Given that (AFAIK) there is no browser which currently supports this out of the box, it seems a little early to be recommending it. @ThunderSon @mackowski what are your thoughts?

Reply:
Definitely not to be discussed _yet_. #95 This issue was opened to discuss trusted types once they are out in the public.